todo_app/data/Trello_items.py

```diff
@@ -0,0 +1,61 @@
+import requests
+import os
+
+from todo_app.data.item import Item
+
+def add_item(title):
+    reqUrl = "https://trello-proxy.azure-api.net/1/cards"
+
+    query_params = {
+        "key": os.getenv("TRELLO_API_KEY"),
+        "token": os.getenv("TRELLO_API_TOKEN"),
+        "idList": os.getenv("TRELLO_TODO_LIST_ID"),
+        "name": title
+    
+    }
+
+
+    response = requests.post(reqUrl, params= query_params)
+
+    print(response.text)
```

Comment:
This sort of print statement can be really useful for debugging, but a potential source of data leaking in production - this might contain details about the new item for example, which could easily have personal data (think to-dos like "go to doctor to talk about <x>", "call person on <number>"!) and means our logs need to be handled as sensitively as our production database

For that reason I'd usually avoid leaving such open/generic print statements in after it's working (sometimes we'll leave them in so they execute in debug mode but not prod, which we'll talk about later!)